esp/esp/program/modules/base.py

```diff
@@ -483,35 +456,8 @@ def update_props(props):
             
         return props
                 
-        
-            
-    @classmethod
-    def getSummary(cls):
-        """
-        Return  the name of a template file that renders the myESP view for this class.
-        Returns None if no such view exists for this class.
-
-        This is a stub, to be overridden by subclasses.
-        """
-        return None
-
-    @classmethod
-    def prepareSummary(cls, context):
-        """
-        Modifies the 'context' dictionary by adding any data that the template pointed to
-        by 'getSummary', needs in its context in order to render proprerly.
-
-        Keys added to 'context' should be strings that contain an identifier that's
-        unique to this class, such as self.__name__.  This is not strictly enforced, though.
-
-        Returns the modified context.
-
-        This is a stub, to be overridden by subclasses.
-        """
-        return context
-
-        class Meta:
-            app_label = 'modules'
+    class Meta:
```

Comment:
This was previously in under a method and thus effectively didn't exist.  I think we do want to have it, but it is potentially a slight behavior change and I'm not sure if that's correct.